projects/storefrontlib/src/shared/components/list-navigation/pagination/pagination.component.html

```diff
@@ -1,26 +1,12 @@
-<ul class="pagination" *ngIf="showPagination()">
-  <!-- Previous -->
-  <li class="page-item" [ngClass]="{ disabled: onFirstPage() || !hasPages() }">
-    <a class="page-link" (click)="clickPageNo(getPagePrevious())">«</a>
-  </li>
-
-  <!-- Page Index -->
-  <li
-    class="page-item"
-    *ngFor="let page of getPageIndicies(); let i = index"
-    [ngClass]="{ active: onPageIndex(i), disabled: showDots(i) }"
-  >
-    <a class="page-link" *ngIf="showDots(i)">...</a>
-    <a
-      class="page-link"
-      *ngIf="!hidePageIndex(i)"
-      (click)="clickPageNo(i + 1)"
-      >{{ i + 1 }}</a
-    >
-  </li>
-
-  <!-- Next -->
-  <li class="page-item" [ngClass]="{ disabled: onLastPage() || !hasPages() }">
-    <a class="page-link" (click)="clickPageNo(getPageNext())">»</a>
-  </li>
-</ul>
+<a
```

Comment:
Since this component is not CMS but rather hard-coded in other components is it possible to modify this template? 
Maybe there should be outlets are something of the sort. What do you think?

Reply:
I'd like to wait for our generic outlet approach. If that will come in 2.0, we should change it right away.